safir/src/safir/schema_manager/_config.py

```diff
@@ -0,0 +1,52 @@
+from typing import TypedDict
+
+from pydantic import AnyUrl, Field
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+__all__ = [
+    "SchemaManagerSettings",
+    "SchemaRegistryClientParams",
+    "SchemaRegistryConnectionSettings",
+]
+
+
+class SchemaRegistryClientParams(TypedDict):
+    """Kwargs used to construct an AsyncSchemaRegistryClient."""
+
+    url: str
+
+
+class SchemaRegistryConnectionSettings(BaseSettings):
+    """Settings for connecting to a confluent-compatible schema registry."""
```

Comment:
At this point, there's no case where anyone would want to use the registry without going though the manager. I combined the settings.

To that point, I also made a convenience method on the settings class to make a `PydanticSchemaManager`, because there's no way to construct one any other way from the settings.

Reply:
Oh, great, thank you!